types/powerapps-component-framework/powerapps-component-framework-tests.ts

```diff
@@ -0,0 +1,88 @@
+// tslint:disable-next-line:no-empty-interface
```

Comment:
Please remove these disabling of lint rules

Reply:
Removed